code/src/components/Game.js

```diff
@@ -0,0 +1,112 @@
+import React, { useEffect } from 'react';
+
+import { useDispatch, useSelector } from 'react-redux'
+
+import { rooms } from '../reducers/rooms'
+import { loader } from '../reducers/loader'
+
+import arrow_one from '../assets/arrow-one.svg'
+import arrow_split from '../assets/arrow-split.svg'
+
+export const Game = () => {
+  const username = useSelector(store => store.rooms.username)
+  const room = useSelector(store => store.rooms)
+  // We'll use this variable to check if things are loading or not.
+  const isLoading = useSelector(store => store.loader.isLoading)
+  const dispatch = useDispatch();
+
+  const START_URL = "https://wk16-backend.herokuapp.com/start";
+  const ACTION_URL = "https://wk16-backend.herokuapp.com/action";
+
+  const startGameFetchInfo = {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({
+      username: username
+    })
+  }
+
+  // This function starts the game, by doing the first fetch from the API.
+  const startGame = () => {
+    dispatch(loader.actions.setLoading(true))
+    // Fetcha, och sedan skicka in responsen (data) till setGameState. Typ.
+    fetch(START_URL, startGameFetchInfo)
```

Comment:
Interesting to see this approach where you initialise the post request before the thunk is made and then you use it in the fetch here 👍

Reply:
Well, it's interesting, but also not reusable - the main reason to use Redux thunks is to make them reusable. Right now, your logic will have to implemented multiple times if you would decide to develop more features. Therefore, let's move those methods to thunks